dune_client/api/base.py

```diff
@@ -179,15 +179,22 @@ def _get(
             return response
         return self._handle_response(response)
 
-    def _post(self, route: str, params: Optional[Any] = None) -> Any:
+    def _post(
+        self,
+        route: str,
+        params: Optional[Any] = None,
+        data: Optional[IO[bytes]] = None,
+        headers: Optional[Dict[str, Optional[str]]] = None,
```

Comment:
Nit: If this defaults to {} and we remove `Optional` we don't need to do the `or {}` further down.

Reply:
Done 👍